src/main/java/salesforce/ui/pages/opportunity/NewOpportunityPage.java

```diff
@@ -0,0 +1,130 @@
+package salesforce.ui.pages.opportunity;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+public class NewOpportunityPage extends BasePage {
+
+    @FindBy(xpath = "//input[@name='IsPrivate']")
+    private WebElement privateCheckBox;
+
+    @FindBy(xpath = "//input[@name='CloseDate']")
+    private WebElement closeDate;
+
+    @FindBy(xpath = "//textarea[@class='slds-textarea']")
+    private WebElement textAreaInput;
```

Comment:
This locator can cause you problems in the future if they decide to add another Textarea field. Besides, if you are using it only for the description the webelement should have that name